### PR TITLE
Add charts for TCPv4/TCPV6/ICMP errors in windows

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
- Add charts for network errors under `IPv4 Networking` and `IPv6 Networking`
- Code cleanup, use macros for code readability
- Further work to make charts "common" as much as possible

### CODE REVIEW

File: src/collectors/windows.plugin/perflib-network.c [hunk 1]
```diff
@@ -137,6 +223,107 @@ struct network_protocol {
...
+            .OutDiscards = { .key = "Datagrams Outbound Discarded" },
+            .OutNoRoutes = { .key = "Datagrams Outbound No Route" },
+            .InAddrErrors = { .key = "Datagrams Received Address Errors" },
+            .InHdrErrors = { .key = "Datagrams Received Header Errors" },
+            .InUnknownProtos = { .key = "Datagrams Received Unknown Protocol" },
+            .type = "ipv4",
+            .id = "errors",
+            .family = "errors",
+            .context = "ipv4.errors",
+            .title = "IPv4 TCP errors",
```
Comment (ilyam8) [inline src/collectors/windows.plugin/perflib-network.c:239]: Why TCP? I think these are just IPv4 error (the same goes for IPv6)

Comment (ilyam8) [reply src/collectors/windows.plugin/perflib-network.c:239]: **If these are v4 (v6)** TCP packets you name metric `ipv4.tcppackets`/`ipv4.tcperrors` => they go into the "Network->IPv4->Protocol->TCP" section.

Comment (thiagoftsm) [reply src/collectors/windows.plugin/perflib-network.c]: @stelfrag, please, take a look in this last comment.

**Comment by thiagoftsm** [comment]
@stelfrag your PR is working as expecting, I am waiting for your comment above or another commit to approve it.

**Comment by stelfrag** [comment]
> @stelfrag your PR is working as expecting, I am waiting for your comment above or another commit to approve it.

It is ok. The title was fixed so it is not TCP related

**Review by thiagoftsm — APPROVED** [review]
The PR is working as expected, LGTM!

**Pull request merged**